### PR TITLE
Tweak burst timestamp design

### PR DESCRIPTION
**Pull request by daehn**
# What's in this PR?

* Extract burst timestamp view and update design.

### CODE REVIEW

**Comment by mikeger** [comment]
@daehn  testThatItRendersMessageWithDayTimestamp_Unread_{320, 102.5}@2x.png - thick line by design?

File: Wire-iOS/Resources/Classy/stylesheet.cas [hunk 1]
```diff
@@ -778,18 +778,26 @@ MessageToolboxView {
...
-    }
+    burstTimestampView: @{
+        label: @{
+            textColor: $color-text-foreground;
+        }
 
-    burstLabelSeparatorView: @{
-        backgroundColor: $color-separator;
+        leftSeparator: @{
+            backgroundColor: $color-audio-button-overlay;
```
Comment (mikeger) [inline Wire-iOS/Resources/Classy/stylesheet.cas]: `color-audio-button-overlay` does not sound like fitting well semantically here. May be you can use some other color with the same value? I believe we have something like `separator-color`

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConversationCell.m [hunk 2]
```diff
@@ -122,10 +116,11 @@ - (instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSStr
         self.longPressGestureRecognizer = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(handleLongPress:)];
         [self.contentView addGestureRecognizer:self.longPressGestureRecognizer];
 
-        
+        @weakify(self);
         self.accentColorChangeHandler = [AccentColorChangeHandler addObserver:self handlerBlock:^(UIColor *newColor, ConversationCell *cell) {
+            @strongify(self);
             cell.tintColor = newColor;
-            cell.unreadDotView.backgroundColor = newColor;
+            self.burstTimestampView.unreadDot.backgroundColor = newColor;
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConversationCell.m]: Should the burst timestamp observe the accent color on it's own?

**Comment by mikeger** [comment]
@daehn Oh, I see, it's by design. Looks a bit like 95' is back, but let's see

File: Wire-iOS/Resources/Classy/stylesheet.cas (hunk 1 above)
Comment (daehn) [inline Wire-iOS/Resources/Classy/stylesheet.cas]: Yeah, was thinking the same. `separator-color` is a different color, but I can change the name of `audio-button-overlay` or add a new one.

**Comment by daehn** [comment]
@mikeger updated!

**Pull request merged**